libraries/Microsoft.Bot.Connector/Authentication/JwtTokenValidation.cs

```diff
@@ -62,22 +62,28 @@ public static async Task<ClaimsIdentity> AuthenticateRequest(IActivity activity,
             if (string.IsNullOrWhiteSpace(authHeader))
```

Comment:
This change also (unfortunately but temporarily) needs to go into ParameterizedBotFrameworkAuthentication.JwtTokenValidation_AuthenticateRequestAsync()
doesn't it?

Reply:
Done, does it look OK now?